components/event-gateway/internal/apperrors/apperrors.go

```diff
@@ -0,0 +1,53 @@
+package apperrors
```

Comment:
Type apperrors was intruduced but it is referenced in apperrors_test.go

Reply:
Done.